controllers/fenceagentsremediation_controller_test.go

```diff
@@ -182,18 +205,74 @@ func getFenceAgentsRemediation(nodeName, agent string, sharedparameters map[v1al
 	}
 }
 
-// buildFarPod builds a dummy pod with FAR label and namespace
-func buildFarPod() *corev1.Pod {
-	fenceAgentsPod := &corev1.Pod{}
-	fenceAgentsPod.Labels = faPodLabels
-	fenceAgentsPod.Name = "mock-fence-agents"
-	fenceAgentsPod.Namespace = defaultNamespace
+// buildPod builds a dummy pod
+func buildPod(containerName, podName, nodeName string) *corev1.Pod {
+	pod := &corev1.Pod{}
+	pod.Name = podName
+	if podName == farPodName {
+		// only when we build FAR pod then we add it's label
+		pod.Labels = faPodLabels
+	} else {
+		// testedPod should be reside on unhealthy node
+		pod.Spec.NodeName = nodeName
+	}
+	pod.Namespace = defaultNamespace
 	container := corev1.Container{
-		Name:  "foo",
+		Name:  containerName,
 		Image: "foo",
 	}
-	fenceAgentsPod.Spec.Containers = []corev1.Container{container}
-	return fenceAgentsPod
+	pod.Spec.Containers = []corev1.Container{container}
+	return pod
+}
+
+// createRunningPod builds new pod format, create it, and set it's status as running
```

Comment:
typo: *its

also below and in `far_e2e_test.go`

Reply:
I used it's when I wanted to use its... I will add that (and other typo comments) in a follow up PR if you don't mind